runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/getPotentialCapabilities/gpc001.c

```diff
@@ -153,6 +153,11 @@ Java_com_ibm_jvmti_tests_getPotentialCapabilities_gpc001_verifyOnLoadCapabilitie
 		unavailableCount--;
 	}
 
+	/* GC policies Metronome & Balanced don't support this event */
+	if (!initialCapabilities.can_generate_sampled_object_alloc_events) {
+		unavailableCount--;
```

Comment:
We greatly prefer `-= 1`

Reply:
Updated this and other occurrences in same method.